CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,72 @@
+# Changelog 2.2
+
+## 2.2.0 - 2024-10-30
+
+### Added
+
+- Add a new feature `PacketCapture` to allow users to capture live traffic and upload captured packets to a specified location:
+  - Add PacketCapture API. ([#6257](https://github.com/antrea-io/antrea/pull/6257), [@hangyan])
+  - Add PacketCapture data path support. ([#6756](https://github.com/antrea-io/antrea/pull/6756), [@hangyan])
+  - Refer to [this document](https://github.com/antrea-io/antrea/blob/release-2.2/docs/packetcapture-guide.md) for more information about this feature.
+- Add a few new antctl sub-commands for `BGPPolicy` feature to improve usability:
```

Comment:
```suggestion
- Add a few new antctl sub-commands for feature `BGPPolicy` to improve usability:
```

Reply:
I feel "the `BGPPolicy` feature" reads better.

Reply:
I agree